lib/motor/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Motor
-  VERSION = '0.4.25'
+  VERSION = '0.4.27'
```

Comment:
This is the correct version, for some reason the version change from my last PR got lost

Reply:
run bundle install in motor folder.... should update the gemlock too